man/groupingsets.Rd

```diff
@@ -55,6 +60,11 @@ rollup(DT, j = sum(value), by = c("color","year","status")) # default id=FALSE
 rollup(DT, j = sum(value), by = c("color","year","status"), id=TRUE)
 rollup(DT, j = lapply(.SD, sum), by = c("color","year","status"), id=TRUE, .SDcols="value")
 rollup(DT, j = c(list(count=.N), lapply(.SD, sum)), by = c("color","year","status"), id=TRUE)
+rollup(DT, j = sum(value), by = c("color","year","status"),
+       label = list(color = "total", year = as.Date("3000-01-01"), status = factor("total"))) # specify label by variable name
+rollup(DT, j = sum(value), by = c("color","year","status"),
+       label = list(color = "total", Date = as.Date("3000-01-01"), factor = factor("total"))) # specify label by variable name and first element of class
+rollup(DT, j = sum(value), by = c("color","year","status"), label = "total") # label is character scalar so applies to color only
```

Comment:
I would like also examples of using NA and NaN in label arg

Reply:
I've added these.